ci_framework/roles/reproducer/tasks/epel.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Enable CRB repositories (CentOS)
+  when:
+    - ansible_facts['distribution'] == 'CentOS'
+  become: true
+  ansible.builtin.command:
```

Comment:
I hoped to find this as a module, but it doesn't exist :(
https://github.com/ansible/ansible/issues/46963

Reply:
don't bother on that file